### DISCUSSION TITLE
3004.388.8_0-gnuton0_alpha1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#621] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: July 05, 2024
Release: 3004.388.8_1-gnuton1

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3004.388.8_0-gnuton0_alpha1'>3004.388.8_0-gnuton0_alpha1</a>.</em>

**Comment by Yakumounr** [comment]
As there is nothing in the gnuton changelog currently, for anyone similarly curious the corresponding Merlin release which this would be based on has it's changlog [here ](https://www.snbforums.com/threads/asuswrt-merlin-3004-388-8-beta-is-now-available.90713/).

**Reply by gnuton** [reply]
@Yakumounr thanks for bringing this up. I will try to find a solution to be more transparent on what has been changed
The complete changelog is here https://github.com/gnuton/asuswrt-merlin.ng/blob/master/Changelog-NG.txt

**Reply by KristianLH** [reply]
@gnuton is there any chance that we can get a fix for the AiMesh issue that we are facing on XT8 (RT-AX95Q)? I seem to be stuck on 3004.388.6_0-gnuton1 since AiMesh in later versions have issues with stable uplink?
Best regards and thank you for all your hard work! 
/Kristian

**Comment by Hyphaed** [comment]
3004.388.8_2 released 6 hours ago

https://github.com/RMerl/asuswrt-merlin.ng/releases/tag/3004.388.8_2

3004.388.8_2 (31-July-2024)
  - UPDATED: OpenVPN to 2.6.12.
  - CHANGED: Support importing WiregUard config files that
             contain multiple AllowedIPs, Address or DNS
              declarations.
  - FIXED: OpenVPN client routing not working properly when
           configuring Internet redirection to "All" or "None".
  - FIXED: New firmware check button missing for the RT-AX58U.
  - FIXED: Generated web certificate wasn't using the FQDN
           for Namecheap DDNS users.

**Reply by meatballvegans** [reply]
Hello, I really want to use a VPN killswitch feature with openvpn configuration file, which is not available on my RT-AX82U. I tried using 3004.388.8_0-gnuton0_alpha1 and I cannot import .ovpn file. There is always an error; "Disconnected" and system log says resetting to default.

I have now installed 3004.388.8_2-gnuton1 and still cannot use .ovpn. Although, wireguard config file seems to upload successfully.  I want to be able to subnet my network and apply vpn infusion to specific subnets with a killswitch feature.
